src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1136,9 +1144,7 @@ public static Map<String, MethodEntry> getMethodMap(String inName) {
    * @return list of registrations
    */
   synchronized public List<Registration> getServiceList() {
-    return registry.values().stream()
-            .map(si -> new Registration(si.getId(), si.getName(), si.getTypeKey()))
-            .collect(Collectors.toList());
+    return registry.values().stream().map(si -> new Registration(si.getId(), si.getName(), si.getTypeKey())).collect(Collectors.toList());

```

Comment:
Why all on one line? It looks a bit harder to read to me

Reply:
auto formatting ... @kwatters and I have share a format file in eclipse
mrl_formatter.xml